sklearn/naive_bayes.py

```diff
@@ -30,7 +30,7 @@
 from .utils.multiclass import _check_partial_fit_first_call
 from .utils.validation import check_is_fitted, check_non_negative
 from .utils.validation import _check_sample_weight
-
+from numbers import Real
```

Comment:
Please place this import in the first group of imports where we import from the std library

Reply:
I have placed the `from numbers import Real` below the std module import of `abc` module.